climada/util/api_client.py

```diff
@@ -844,6 +844,44 @@ def get_litpop_default(self, country=None, dump_dir=SYSTEM_DIR):
             raise ValueError("country must be string or list of strings")
         return self.get_exposures(exposures_type='litpop', dump_dir=dump_dir, properties=properties)
 
+    def get_centroids(self, res_arcsec_land=150, res_arcsec_ocean=1800, extent=None, reg_id=None, country_iso3alpha=None,
+                      dump_dir=SYSTEM_DIR):
+        """Get global centroids
+
+        Parameters
+        ----------
+        res_land_arcsec : int
+            resolution for land centroids in arcsec. Default is 150
+        res_ocean_arcsec : int
+            resolution for ocean centroids in arcsec. Default is 1800
+        reg_id : int
+            region to filter according to region_id values
+        extent : tuple
+            Format (min_lon, max_lon, min_lat, max_lat) tuple.
+            If min_lon > lon_max, the extend crosses the antimeridian and is
+            [lon_max, 180] + [-180, lon_min]
+            Borders are inclusive.
+        dump_dir : str
+            directory where the files should be downoladed. Default: SYSTEM_DIR
+        Returns
+        -------
+        climada.hazard.centroids.Centroids
+            global centroids from the api
+        """
+
+        res_arcsec_land = str(res_arcsec_land)
+        res_arcsec_ocean = str(res_arcsec_ocean)
+        extent_property = '(-180, -90, 180, 90)'
+        dataset = self.get_dataset_info("centroids", properties={'res_arcsec_land':res_arcsec_land,
+                                                                 'res_arcsec_ocean':res_arcsec_ocean,'extent':extent_property})
```

Comment:
Maybe a check whether the data exist would be useful? Because the error message from `get_dataset_info` is not very useful if for instance I say `res_arcsec_land=2500` and it does not exist...